tests/console/sudo.pm

```diff
@@ -105,8 +105,9 @@ sub run {
     zypper_call 'in sudo expect';
     select_console 'user-console';
     # Defaults targetpw -> asks for root PW
-    my $exp_user = (is_azure && is_sle('>=15-SP4')) ? 'bernhard is not in the sudoers file' : 'root';
-    validate_script_output("expect -c 'spawn sudo id -un;expect password {send \"$testapi::password\\r\";interact}'", sub { qr/^$exp_user\$/ });
+    my $exp_user = (is_azure && is_sle('>=15-SP4')) ? 'bernhard' : 'root';
```

Comment:
Our tests should always consider openSUSE Tumbleweed as the default product without a version. That means any differing behaviour should have an explicit exclude rule for the "older products" and potentially an explanation why the behaviour should differ. You can try to negate the logic check.

See https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md#coding-style for more details

Also a verification run on Tumbleweed would be nice

Reply:
There are no Public Cloud images for Tumbleweed.

Reply:
True but the part I was changing is very minimal and PublicCloud only - even more specifically - Azure only therein.

I considered the 15-SP5 GCE test run sufficient to exclude unwanted side-effects but if wanted I can schedule a Tumbleweed test run as well.